### DISCUSSION TITLE
Map data

### DISCUSSION THREAD

**Discussion by Naybs808** [rinigus/osmscout-server#431] [General]
I got the impression from tinkering around that the server uses mapbox tiles to display map data. Is that right? Is the map data open source, how is it licensed?

**Comment by Naybs808** [comment]
Ahh found in the README. 

Maps are hosted by Natural Language Processing Centre
(https://nlp.fi.muni.cz/en/ , Faculty of Informatics, Masaryk
University, Brno, Czech Republic) through modRana (http://modrana.org)
data repository. Maps are updated roughly every 2-3 months.

Map data from OpenStreetMap, Open Database License 1.0. Maps are
converted to a suitable format from downloaded extracts and/or using
polygons as provided by Geofabrik GmbH.

UK postal codes downloaded from https://www.freemaptools.com. These
data contain: Ordnance Survey data © Crown copyright and database
right 2017; Royal Mail data © Royal Mail copyright and database right
2017; National Statistics data © Crown copyright and database right
2017.

**Comment by kaelte** [comment]
Is it possible to use cycling or hiking maps offline via OSM Scour Server.
I would like to have maps similar to https://www.opencyclemap.org/ or https://opentopomap.org
